.github/workflows/contracts.yml

```diff
@@ -197,7 +194,6 @@ jobs:
       # `etherscan-verify` plugins tries to verify them, which is not desired.
       - name: Prepare for verification on Etherscan
         run: |
-          rm -rf ./node_modules/@keep-network/keep-core
           rm -rf ./external/npm
```

Comment:
We can remove this whole step (the `rm -rf ./external/npm` become obsolete some time ago).

Reply:
✔️